commonlib/src/main/java/com/crisolutions/commonlib/ActivityLifecycleTask.java

```diff
@@ -0,0 +1,34 @@
+package com.crisolutions.commonlib;
+
+import android.support.v7.app.AppCompatActivity;
+
+public abstract class ActivityLifecycleTask {
```

Comment:
Maybe we can revise this in a later issue/PR. But, I wonder if this could be deprecated in favor of `LifecycleObserver`.

Reply:
Noted, I'll discuss that with the other folks as well to make sure.